.env

```diff
@@ -0,0 +1,2 @@
+PORT=3000
```

Comment:
you should never push your .env file. add it ot gitignore. you can push an example.env after you are done so people know the variables to add but in the meantime add this to gitignore